modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js

```diff
@@ -129,14 +128,18 @@ export const FormBuilder = () => {
 				({fieldName}) => fieldName === 'requireAuthentication'
 			);
 
+			const formInstanceId = document.querySelector(
+				`#${portletNamespace}formInstanceId`
+			).value;
+
```

Comment:
@mlan0, did you need this code because the `formInstanceId` from [here](https://github.com/liferay/liferay-portal/blob/26c6bc1b470b21c0ea00632554536f3a9ad8e17b/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js#L48) was `undefined`? Can you please give me more details?

Thanks.

Reply:
Yes, the formInstanceId had a value of 0, as the form wasn't saved. And then, if there was just the change to save the form before getting the preview URL, it still wouldn't be enough, as the formInstanceId would not have been updated in time.

Reply:
Let's talk about this pull after our daily meeting today. Can you please remind me about this pull today?

Thanks.